### PR TITLE
[ACCESSIBILITY/CI/CD] Update letters e2e tests

### PR DESCRIPTION
**Pull request by Mottie**
## Description

The letters app end-to-end tests did not previously expand all content accordions and perform an accessibility check using axe. This PR does that.

## Testing done

Updated e2e test

## Screenshots

Screen Shot 2020-04-10 at 2 27 52 PM

## Acceptance criteria
- [x] Axe check is performed on letters app expanded accordions

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by micahchiang — APPROVED** [review]
lgtm 👍

**Comment by Mottie** [comment]
Huh, @micahchiang I'm not sure why your review didn't count toward a review :(

**Review by ndsprinkle — APPROVED** [review]

**Comment by ncksllvn** [comment]
@Mottie The codeowners file for Letters has -

```
src/applications/letters @department-of-veterans-affairs/vsa-bam-1-frontend @department-of-veterans-affairs/vsa-authd-exp-frontend
```

So it's owned by BAM1 and Auth Exp. Should it be changed to BAM1 and eBenefits? Or we can just add eBenefits to the list. Feel free to edit the codeowners file and open a PR 🙂

File: src/applications/letters/tests/01-authed.e2e.spec.js
Comment (ncksllvn) [inline src/applications/letters/tests/01-authed.e2e.spec.js:27]: Oh cool, I couldn't get the `elements` method to work but now I see I wasn't adding that `'css selector'` part.

**Review by ncksllvn — APPROVED** [review]
Great work!

**Comment by Mottie** [comment]
> Should it be changed to BAM1 and eBenefits?

LOL I have no idea who else owns this app, but it's fine now. I'm just waiting for an accessibility review to merge it 😺

File: src/applications/letters/tests/01-authed.e2e.spec.js
Comment (Mottie) [reply src/applications/letters/tests/01-authed.e2e.spec.js:27]: Yeah, nightwatch is new to me too 🤣

**Review by 1Copenut — APPROVED** [review]
💯I dig this new test flow and using the `elements` selector. I ran the test locally and verified SC from issue ticket 7385.

**Review by jenstrickland — APPROVED** [review]

**Pull request merged**
